docs/specification/draft/basic/authorization.mdx

```diff
@@ -304,3 +304,28 @@ Attackers can exploit MCP servers acting as intermediaries to third-party APIs,
 
 MCP proxy servers using static client IDs **MUST** obtain user consent for each dynamically
 registered client before forwarding to third-party authorization servers (which may require additional consent).
+
+### 3.5 Access Token Privilege Restriction
+
+An attacker can gain unauthorized access or otherwise compromise a MCP server if the server accepts tokens issued for other resources.
+
+This vulnerability has two critical dimensions:
+
+1. **Audience validation failures.** When an MCP server doesn't verify that tokens were specifically intended for it (for example, via the audience claim, as mentioned in [RFC9068](https://www.rfc-editor.org/rfc/rfc9068.html)), it may accept tokens originally issued for other services. This breaks the fundamental OAuth security boundary, allowing attackers to reuse legitimate tokens across different services than intended.
+2. **Token passthrough.** If the MCP server not only accepts tokens with incorrect audiences but also forwards these unmodified tokens to downstream services, it introduces the "confused deputy" risk, outlined in [Section 3.4](#34-confused-deputy-problem). The MCP server becomes an unintentional proxy that can:
+  1. Enable privilege escalation across service boundaries
+  1. Facilitate lateral movement through connected resources
+  1. Allow circumvention of security controls between services
+  1. Enable replay attacks against multiple backend systems
+
+MCP servers **MUST** take all necessary steps to ensure no data is returned to unauthorized parties and **MUST** ensure any credentials are valid before processing the request.
```

Comment:
```suggestion
MCP servers **MUST** validate access tokens before processing the request, and take all necessary steps to ensure no data is returned to unauthorized parties.
```

Are there any credentials expected other than an access token? Seems better to be explicit here.